AB_Hands_On/AB_Hands_On.ipynb

```diff
@@ -0,0 +1,6511 @@
+{
```

Comment:
<p>These read well here but they seem long. Lets double in slide format they dont have a line break, otherwise itll be too dense to read.</p><p> </p><p>If there is a line break use sub bullets</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>